lib/src/emojis/emojis.dart

```diff
@@ -60,15 +60,14 @@ class Emojis {
   /// Returns the Emoji with the specified value, unicode, name, or shortcode.
   /// If the specified Emoji cannot be found, the orElse parameter is called, and its
   /// return value is returned. If orElse is not specified, a [StateError] is thrown.
-  Emoji getOne(String param, {Emoji Function()? orElse}) {
-    return _emojis.firstWhere(
+  Emoji? getOne(String param) {
```

Comment:
There is already a method that covers this:

It's called [getOneOrNull](https://github.com/Nikoro/emoji_extension/blob/417930b81e533023a25ce287ac45e9f7b0877a81/lib/src/emojis/emojis.dart#L77)

So instead of changing this`getOne` method. You could use `getOneOrNull` directly then:

```dart
  String fromShortcodes({String Function(String unknownShortcode)? onUnknownShortcode}) {
    return _value.splitMapJoin(Regex.shortcode, onMatch: (m) {
      final match = m.group(0)!;
      return _emojis.getOneOrNull(match.removeColons())?.value ?? onUnknownShortcode?.call(match) ?? match;
    });
  }
```

Reply:
I missed that 👍